Cognite.Simulator.Utils/DefaultClasses/DefaultConnectorRuntime.cs

```diff
@@ -0,0 +1,195 @@
+
+                    await connector.Init(token).ConfigureAwait(false);
+                   
+                    await connector.Run(token).ConfigureAwait(false);
+                }
+                catch (TaskCanceledException) when (token.IsCancellationRequested)
+                {
+                    logger.LogWarning("Connector manually interrupted. Exiting...");
+                    break;
+                }
+                catch (Exception e)
+                {
+                    await pipeline.NotifyError(e, token).ConfigureAwait(false);
+                    if (e is ConnectorException ce)
+                    {
+                        logger.LogError("Connector error: {Message}", ce.Message);
+                        if (ce.InnerException != null)
+                        {
+                            logger.LogError("- {Message}", ce.InnerException.Message);
+                        }
+                        foreach (var err in ce.Errors)
+                        {
+                            logger.LogError("- {Message}", ce.Message);
+                        }
+                    }
+                    else if (e is CogniteSdk.ResponseException re)
+                    {
+                        // CDF request failed but request id and status code are available.
+                        logger.LogError("Request to CDF failed with code {Code}: {Message}. Request id: {RequestId}", re.Code, re.Message, re.RequestId);
+                    }
+                    else if (e is HttpRequestException he)
+                    {
+                        // Did not get response from server (server down or other networking errors).
+                        logger.LogError("The HTTP client failed to contact CDF: {Message}", he.Message);
+                    }
+                    else if (e is JsonException je)
+                    {
+                        // Json decode exception produced by the SDK. Unlikely to happen.
+                        logger.LogError("Response from CDF cannot be parsed: {Message}", je.Message);
+                    }
+                    else if (e is TimeoutRejectedException)
+                    {
+                        // Timeout from the http client policy.
+                        logger.LogError("Request to CDF timeout after retry");
+                    }
+                    else if (e is SimulatorConnectionException sue)
+                    {
+                        // If the simulator is not available, log to Windows events and exit
+                        defaultLogger.LogError(sue.Message);
+                        logger.LogError(sue, sue.Message);
+                        break;
+                    }
+                    else
+                    {
+                        logger.LogError(e, "Unhandled exception: {message}", e.Message);
+                        // sourceProgram.Cancel();
```

Comment:
Can we find out what this commented code does?

Reply:
Seems to be part of old dead code (the variable does not exist), so deleted it